app/main/server/preferences/schemas/window-windows.json

```diff
@@ -0,0 +1,77 @@
+{
+  "type": "object",
+  "properties": {
+    "windowDraggable": {
+      "type": "boolean",
+      "title": "Draggable window",
+      "default": true
+    },
+    "rememberWindowPosition": {
+      "type": "boolean",
+      "title": "Automatically remember window position",
+      "default": false
+    },
+
+    "position": {
+      "type": "object",
+      "title": "Position",
+      "properties": {
+        "positionWindow": {
+          "type": "string",
+          "title": "Position mode",
+          "default": "center",
+          "enum" : [
+            [
+              "center",
+              "Center window on screen"
+            ],
+            [
+              "specified",
+              "Set window to specific screen position"
+            ]
+          ]
+        },
+
+        "posX": {
+          "condition": ".position.positionWindow=='specified'",
```

Comment:
@appetizermonster I tried a couple of different approaches but couldn't work out the best way to achieve this in the rendering code, but you can see what I'd like to achieve here.

I'd like some of these items to only be visible under specific conditions (as noted).

Is this something you'd be able to implement?
The feature is OK without it, but it'd make it nicer!

Reply:
How do you think about this shape of query like mongodb?

```json
{
  "condition": {
    "$and": [
      ".position.positionWindow": {
        "$eq": "specified"
      },
      ".position.x": {
        "$gte": 0.5
      }
    ]
  }
} 
```

Reply:
I actually took the condition format from the docs for the existing json-schema docs (which we seem to be implementing?):

https://github.com/json-schema-form/json-schema-form/wiki/Documentation#condition